app/forms/trainee_id_form.rb

```diff
@@ -3,11 +3,7 @@
 class TraineeIdForm < TraineeForm
   attr_accessor :trainee_id
 
-  validates :trainee_id, presence: true,
-                         length: {
```

Comment:
Where did the length validation go?

Reply:
I removed it by accident thinking it was meant to be a foreign key value - forgot that it was a free text field. It was a separate commit so I just had to drop it. Goot spot.